examples/trials/cifar-adaptdl/main.py

```diff
@@ -124,21 +116,18 @@ def train(epoch):
         stats["total"] += targets.size(0)
         stats["correct"] += predicted.eq(targets).sum().item()
 
-        writer.add_scalar("Throughput/Gain", net.gain, epoch)
-        writer.add_scalar("Throughput/Global_Batchsize",
-                          trainloader.current_batch_size, epoch)
-
+    trainloader.to_tensorboard(writer, epoch, tag_prefix="AdaptDL/Data/")
```

Comment:
Is "to_tensorboard" a new feature? I did not see it in other examples.

Reply:
I see it in the updated AdatptDL example: https://github.com/petuum/adaptdl/blob/master/examples/pytorch-cifar/main.py#L127